packages/matchbox/src/components/ProgressBar/tests/ProgressBar.test.js

```diff
@@ -1,26 +1,59 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-
 import ProgressBar from '../ProgressBar';
+import cases from 'jest-in-case';
+import 'jest-styled-components';
 
-describe('ProgressBar', () => {
-  let wrapper;
+const styleCases = [
+  {
+    name: 'default progress bar',
+    props: {
+      completed: 80,
+    },
+    assert: [
+      ['height', '0.75rem'],
+      ['background-color', '#d9e0e6'],
+    ],
+  },
+  {
+    name: 'normal size',
+    props: {
+      completed: 80,
+      size: 'normal',
+    },
+    assert: [['height', '0.75rem']],
+  },
+  {
+    name: 'small size',
+    props: {
+      completed: 80,
+      size: 'small',
+    },
+    assert: [['height', '0.25rem']],
+  },
+];
 
-  beforeEach(() => {
-    wrapper = shallow(<ProgressBar />);
-  });
+cases(
+  'renders progress bar styles',
+  ({ props, assert }) => {
+    const wrapper = global.mountStyled(<ProgressBar {...props} />);
 
-  it('renders progressbar at defaults', () => {
-    expect(wrapper).toMatchSnapshot();
-  });
+    if (!Array.isArray(assert[0])) {
+      expect(wrapper).toHaveStyleRule(assert[0], assert[1]);
+    } else {
+      for (const assertion of assert) {
+        expect(wrapper).toHaveStyleRule(assertion[0], assertion[1]);
+      }
+    }
+  },
+  styleCases,
+);
 
-  it('renders at 50% progress in navy color', () => {
-    wrapper.setProps({ completed: 50, color: 'navy' });
-    expect(wrapper).toMatchSnapshot();
-  });
+describe('renders progress bar inner styles', () => {
+  const progressBar = props => global.mountStyled(<ProgressBar {...props} />);
+
+  it('should render progress bar width based on completed percent', () => {
+    const wrapper = progressBar({ completed: 24 });
 
-  it('renders at 100% progress if given a value over 100', () => {
```

Comment:
It would be worth leaving a test in for the `completed` value exceeding `100` and going below `1` (maybe even a negative value) just to make sure we are keeping track of that requirement that the bar can't go below 0 or above 100.